internal/storage/postgres_test.go

```diff
@@ -0,0 +1,97 @@
+package storage
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestPGMeta(t *testing.T) {
+	dummyPgMeta := pgmeta{
+		cols: map[string][]string{
+			"table1": []string{"0", "jason", "big ben"},
+			"table2": []string{"1", "john", "bakers street"},
+			"table3": []string{"2", "harry", "leicester square"},
+		},
+		pks: map[string][]string{
+			"table1": []string{"id"},
+			"table2": []string{"id", "name"},
+			"table3": []string{"id", "name", "address"},
+		},
+		bytes: map[string]int64{
+			"table1": 1234,
+			"table2": 2345,
+			"table3": 10000,
+		},
+	}
+
+	ctx := context.Background()
+	pdb, err := NewPostgres(ctx, "postgres://postgres:localhost:5432")
+	if err != nil {
+		t.Fatalf("failed to create postgres client: %v", err)
+	}
+	pdb.meta = &dummyPgMeta
+	_, err = pdb.StartTx(ctx)
```

Comment:
We don't need to start a transaction here.

Reply:
Removed it.